ibm_was/tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
 minversion = 2.0
 skip_missing_interpreters = true
-basepython = py27
+basepython = py36
 envlist =
-    ; 8.5
+    ; py{27,36}-8.5
```

Comment:
why is this commented out ?

Reply:
@nmuesch work, i just kept it there

Reply:
@ofek  added E2E with docker containers. For some reason the E2E setup doesn't currently work with 8.5. This + a card on our backlog was created to address this. It was left here so we tackle it soon.